content/docs/user-guide/integrations/sagemaker.md

```diff
@@ -0,0 +1,50 @@
+# Amazon SageMaker
+
+## Setup
+
+Many DVC features rely on Git. To work with DVC in Amazon SageMaker, first setup
+your Git repo:
+
+1. [Clone] a repository.
+
+   ![Git clone](/img/sagemaker-git-clone.gif)
+
+2. Launch a terminal or notebook and configure the user name and email:
+
+   ```bash
+   git config --global user.name ...
+   git config --global user.email ...
+   ```
+
+Don't forget to install DVC and any other requirements in your environment!
+
+## Notebooks
+
+After completing the setup, you can work with DVC in SageMaker notebooks like
+you would in any other environment. Take a look at DVC [experiments] for how to
+get started with DVC in notebooks. If you would like to see live experiment
+updates in [DVC Studio], set your [token].
+
+![Notebook](/img/sagemaker-notebook.gif)
+
+## Pipelines
+
+You can run SageMaker jobs in DVC [pipelines] or convert existing SageMaker
+pipelines into DVC pipelines. This combines the benefits of SageMaker jobs, like
+running each stage on its own EC2 instance and enabling other data
+[input modes](https://docs.aws.amazon.com/sagemaker/latest/dg/model-access-training-data.html),
+with the benefits of DVC pipelines, like skipping unchanged stages and tracking
+the inputs and outputs of each run. SageMaker expects all inputs and outputs to
+be stored in S3, so the easiest way to integrate with DVC is to use [external
+dependencies and outputs]. For an example, see
+https://github.com/iterative/sagemaker-pipeline.
+
```

Comment:
I feel like there should be some explanation and exapmple snippets in this page (from the example repo), to explain the approach / basics of combining of `dvc.yaml` (dvc as pipeline control) and SM as building block. No need to detail too much but maybe a snippet showing what's happening in `sm_prepare.py` (very simple) and linking to `sm_training.py` explaining it showcases a SM based training step wrapped in a script which dvc pipeline runs (and showing it's pipeline stage)

Reply:
I gave it a shot. PTAL 🙏 .